app/src/main/res/xml/appfilter.xml

```diff
@@ -2679,6 +2682,10 @@
     <!-- Transdrone -->
     <item component="ComponentInfo{org.transdroid.lite/org.transdroid.core.gui.TorrentsActivity_}" drawable="transdrone" />
 
+    <!-- Transdroid -->
+    <item component="ComponentInfo{org.transdroid.search/org.transdroid.search.gui.SettingsActivity}" drawable="transdroid"/>
+    <item component="ComponentInfo{org.transdroid.full/org.transdroid.core.gui.TorrentsActivity_}" drawable="transdroid"/>
```

Comment:
It looks like Transdroid is already added five lines above this section, are those incorrect?